code/hispania/game/area/Space Station 13 areas.dm

```diff
@@ -12,3 +12,6 @@
 /area/shuttle/tsf
 	name = "TSF Discovery"
 	icon_state = "shuttle"
+
+/area/centcom/bar
+	name = "Centcom Bar"
```

Comment:
a hispania

Reply:
![image](https://user-images.githubusercontent.com/52227547/116959259-13e22d00-ac6b-11eb-80aa-05b5631980ce.png)

Reply:
sorry, trabajé to er dia y toy más ciego que vivo.